docs/content/tutorial/connecting-to-external-services.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: "Tutorial, part seven: Connecting to external services | Dagster Docs"
+description: Learn how to connect to external services in Dagster with resources
+---
+
+Throughout the tutorial, you have built a data pipeline that connects to different external services, such as Hacker News. By building this asset graph, Dagster acts as a control plane of multiple different external services. However, connecting to these services and managing their use can be difficult. Resources are the recommended way to manage these connections in Dagster.
```

Comment:
```suggestion
Throughout the tutorial, you built a data pipeline that connects to different external services, such as Hacker News. By building this asset graph, Dagster acts as a control plane of multiple different external services. However, connecting to these services and managing their use can be difficult. Resources are the recommended way to manage these connections in Dagster.
```

💭 Suggesting we remove the `have` from `you have built` to make this more active and concise